client/storage.c

```diff
@@ -252,13 +252,11 @@ uint64_t xdag_load_blocks(xdag_frame_t start_time, xdag_frame_t end_time, void *
 			}
 			sum++;
 		}
-
+        
 		if (todo != bufsize) {
 			if (f) {
 				pthread_mutex_lock(&storage_mutex);
-				
-				int res = correct_storage_sums(start_time, &s, 0);
-				
+                int res = correct_storage_sums(start_time, &s, 0);
```

Comment:
Please use tab instead of space. One tab equals to 4 spaces.

Reply:
ok